### PR TITLE
feat - added login and signup page

### PR DESCRIPTION
**Pull request by Balakumar3018**
Added login and sign up page

What to check
  * Just added UI page
  * You can click account in Nav bar and login page will come
  * Click create new  account link it redirect to signup page
  * Click already have account in signup page it redirects to login page
  
Preview - https://cool-trifle-28fdb1.netlify.app/

### CODE REVIEW

**Review by sohan9819 — APPROVED** [review]
Looks good to me . Juts a suggestion to highlight your shop now buttons .

File: src/pages/login-page/LoginPage.jsx [hunk 1]
```diff
@@ -1,5 +1,21 @@
+import "./login-page.css";
+import { Link } from "react-router-dom";
+
 export default function LoginPage(){
     return(
-        <h2>this s from LoginPage</h2>
+        <div className="login-page">
+            <div className="login-card">
+                <h2>Login</h2>
+                <label for="email" >
```
Comment (xii-vi) [inline src/pages/login-page/LoginPage.jsx:9]: use htmlFor instead of for

File: src/pages/login-page/login-page.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    border: 1px  var(--secondary-color);
+    display: flex;
+    flex-direction: column;
+    width: 400px;
+    justify-content: space-evenly;
+    align-content: center;
+    align-items: center;
+    background-color: var(--background-colorWhite);
+    padding: 52px;
+    height: 380px;
```
Comment (xii-vi) [inline src/pages/login-page/login-page.css:11]: _try not to use absolute value_

**Review by xii-vi — APPROVED** [review]
_Left a Few sanity comments. 
Go through them once before merging._

**Pull request merged**